packages/icons/scripts/build.js

```diff
@@ -47,7 +51,12 @@ async function getFiles(src) {
   const files = await getFilesInFolder(src.dir);
   const contents = await Promise.all(
     files
-      .filter(file => file.slice(file.length - 1) !== '.svg')
+      .filter(
+        file =>
+          // file is svg and not in excluded list
+          file.slice(file.length - 1) !== '.svg' &&
```

Comment:
Uhhhh this expression `file.slice(file.length - 1) !== '.svg'` will _always_ be true. Should be able to just do `.endsWith('.svg')` instead, or `file.slice(-4)`.

Reply:
shows that line is entirely unnecessary, will fix though instead of deleting incase that ever changes.